status-im/status-react
# Integrate record audio component into composer

fixes #14896 

### Summary

This PR integrates record audio component into message composer, enabling audio messages again :)
Also added the capability of replying a message with an audio, which it was not possible before (I think), and removed the "Edit message" option for audio messages, because we can't edit an audio.

https://user-images.githubusercontent.com/18485527/222128862-20a3ccb3-1f34-4b38-b16f-7248f1dddc6c.mp4

### Testing notes

- Audio messages needs to be redesigned and are not part of this PR, we have an issue for Audio Message reimplementation: #13530 

#### Platforms

- Android
- iOS

##### Functional

- 1-1 chats
- public chats
- group chats

### Steps to test

- Open Status
- Open any chat
- Tap the mic button to interact with the record audio component and record an audio
- Verify audio messages are being sent correctly

status: ready
